src/lepton/bitops.hh

```diff
@@ -374,6 +374,8 @@ public:
 	bool error;	
 	
 private:
+    void extend_buffer( int newSize );
```

Comment:
Broken indent

Reply:
Yeah, abytereader/abytewriter uses tabs for indent, while other classes in these files uses 4 spaces.